src/main/java/com/cloudbees/groovy/cps/tool/Driver.java

```diff
@@ -78,18 +79,21 @@ public void run() throws Exception {
              *      2. has a Closure parameter in one of the arguments, not in the receiver
              */
             Predicate<ExecutableElement> selector = (e) -> {
-                return e.getKind() == ElementKind.METHOD
-                    && e.getModifiers().containsAll(PUBLIC_STATIC)
-                    && e.getParameters().subList(1, e.getParameters().size()).stream()
-                        .anyMatch(p -> t.types.isAssignable(p.asType(), closureType))
-                    && !EXCLUSIONS.contains(e.getEnclosingElement().getSimpleName().toString()+"."+e.getSimpleName().toString());
+                boolean r = e.getKind() == ElementKind.METHOD
+                        && e.getModifiers().containsAll(PUBLIC_STATIC)
```

Comment:
Further experiments -
```groovy
    @Test
    void overrideStatic() {
        assert evalCPS('''
            public String bar(List<String> l) {
                return bar((Iterable)l)
            }

            public String bar(Iterable<String> l) {
                return "iterable"
            }

            List<String> s = ["a", "b"]

            return bar(s)   
        ''') == "iterable"
    }
```
Does fail the same way. Both the static and non-static variants work fine in non-CPS Groovy.